### DISCUSSION TITLE
Grabación del viernes 07/01/22

### DISCUSSION THREAD

**Discussion by josmanyel** [Launch-X-Latam/MisionFrontEnd#180] [General]
Hola compañeros, no pude estar presente en el envivo del viernes. Pero ayer sábado cuando fui a las grabaciones del Team me encontré con otra grabación... Creo que el MC se equivocó y colgó una grabación de otros becarios de seguridad informática. ¿Alguno se dio cuenta? Le envié un msj a su privado, pero no lo ha visto aún.

**Comment by MacaCoronel93** [comment]
Hola acabo de chequear la clase del viernes es la de html y es la que coincide con la del viernes. Esta correcta fijate que tenes que ingresar donde dice 2 modulo ..

**Reply by xXleonelXxD** [reply]
Disculpa como puedes acceder a oas clases grabadas? Yo quiero verlas pero no sé como

**Reply by josmanyel** [reply]
Cuando entras en el equipo hay un menú y en ese menú encuentras grabaciones. 
![screenshot-teams microsoft com-2022 10 09-20_41_31](https://user-images.githubusercontent.com/114273197/194786968-314d23b1-fda9-4e32-82dc-e4edecaeaef3.png)


> Hola acabo de chequear la clase del viernes es la de html y es la que coincide con la del viernes. Esta correcta fijate que tenes que ingresar donde dice 2 modulo ..

**Reply by josmanyel** [reply]
Si es que me fui directo a grabaciones, pero ya me di cuenta que el link sigue activo y pueda verla desde el link. Pensaba que el link quedaba inactivo, gracias.

**Reply by josmanyel** [reply]
> Hola acabo de chequear la clase del viernes es la de html y es la que coincide con la del viernes. Esta correcta fijate que tenes que ingresar donde dice 2 modulo ..

Gracias bro, es que me fui directo a grabaciones, pero ya me di cuenta que el link sigue activo y pueda verla desde el link. Pensaba que el link quedaba inactivo, gracias.

**Reply by MacaCoronel93** [reply]
Hola soy hermana jajaj no hermano pero me alegro que te haya servido jajaj

**Reply by josmanyel** [reply]
> Hola soy hermana jajaj no hermano pero me alegro que te haya servido jajaj

ahh jaja disculpa pensé que eras chico :-)

**Reply by xXleonelXxD** [reply]
> Cuando entras en el equipo hay un menú y en ese menú encuentras grabaciones. 
> ![screenshot-teams microsoft com-2022 10 09-20_41_31](https://user-images.githubusercontent.com/114273197/194786968-314d23b1-fda9-4e32-82dc-e4edecaeaef3.png)
> 
> 
> > Hola acabo de chequear la clase del viernes es la de html y es la que coincide con la del viernes. Esta correcta fijate que tenes que ingresar donde dice 2 modulo ..
> 
> 

Dale gracias bro

**Comment by iam-leo** [comment]
Que tal explorer? Si tal y como dices hay un error, en la pestaña de grabaciones, hay un video de otro programa de innovaccion. 
Puedes encontrar el link correspondiente a la clase del viernes 07/10 en canal del "Módulo 2 - HTML" como dice @MacaCoronel93 o puedes verlo [aquí](https://web.microsoftstream.com/video/e1369afd-b280-4c18-a687-93e389a50102) (es el canal del MC en Microsoft Streams)

**Reply by xXleonelXxD** [reply]
> Que tal explorer? Si tal y como dices hay un error, en la pestaña de grabaciones, hay un video de otro programa de innovaccion. 
> Puedes encontrar el link correspondiente a la clase del viernes 07/10 en canal del "Módulo 2 - HTML" como dice @MacaCoronel93 o puedes verlo [aquí](https://web.microsoftstream.com/video/e1369afd-b280-4c18-a687-93e389a50102) (es el canal del MC en Microsoft Streams)

Disculpa a mí aún no me asignan grupo y pues no me sale nada de eso aún en teams sabes que puedo hacer?

**Reply by descob4r** [reply]
Tienes que checar el [Onboarding](https://github.com/Launch-X-Latam/OnBoarding) leerlo para que tengas acceso al grupo en teams

**Reply by josmanyel** [reply]
> > Que tal explorer? Si tal y como dices hay un error, en la pestaña de grabaciones, hay un video de otro programa de innovaccion.
> > Puedes encontrar el link correspondiente a la clase del viernes 07/10 en canal del "Módulo 2 - HTML" como dice @MacaCoronel93 o puedes verlo [aquí](https://web.microsoftstream.com/video/e1369afd-b280-4c18-a687-93e389a50102) (es el canal del MC en Microsoft Streams)
> 
> Disculpa a mí aún no me asignan grupo y pues no me sale nada de eso aún en teams sabes que puedo hacer?

Tienes que leerlo paso a paso y hacer todo según cada paso.

**Reply by josmanyel** [reply]
> Que tal explorer? Si tal y como dices hay un error, en la pestaña de grabaciones, hay un video de otro programa de innovaccion. Puedes encontrar el link correspondiente a la clase del viernes 07/10 en canal del "Módulo 2 - HTML" como dice @MacaCoronel93 o puedes verlo [aquí](https://web.microsoftstream.com/video/e1369afd-b280-4c18-a687-93e389a50102) (es el canal del MC en Microsoft Streams)

Gracias bro, es que me fui directo a grabaciones, pero ya me di cuenta que el link sigue activo y pueda verla desde el link. Pensaba que el link quedaba inactivo, gracias.

**Reply by xXleonelXxD** [reply]
> Tienes que checar el [Onboarding](https://github.com/Launch-X-Latam/OnBoarding) leerlo para que tengas acceso al grupo en teams

Ya lo hice ayer en la mañana voy a esperar hasta mañana ya que puede que porque sea fin de semana no me agreguen aún

**Reply by iam-leo** [reply]
> > Que tal explorer? Si tal y como dices hay un error, en la pestaña de grabaciones, hay un video de otro programa de innovaccion.
> > Puedes encontrar el link correspondiente a la clase del viernes 07/10 en canal del "Módulo 2 - HTML" como dice @MacaCoronel93 o puedes verlo [aquí](https://web.microsoftstream.com/video/e1369afd-b280-4c18-a687-93e389a50102) (es el canal del MC en Microsoft Streams)
> 
> Disculpa a mí aún no me asignan grupo y pues no me sale nada de eso aún en teams sabes que puedo hacer?

Te dejo los canales en Microsoft Streams de [Launch X Misión FrontEnd Latam](https://web.microsoftstream.com/channel/811c421f-c2fa-4a13-97ea-e47b9c3fe60b) y del [Mission Commander](https://web.microsoftstream.com/user/96eeb66f-b984-4083-8ea6-74c34a07a34a) en ambos canales están las transmisiones de las clases, creería que no tendrás problema en acceder, solo tienes que ingresar con tu cuenta de innovaccion, pero como dicen todos lo importante aquí es que puedas hacer tu onboarding y si no puedes contáctate con el equipo de innovaccion lo antes posible!

**Reply by iam-leo** [reply]
> > Que tal explorer? Si tal y como dices hay un error, en la pestaña de grabaciones, hay un video de otro programa de innovaccion. Puedes encontrar el link correspondiente a la clase del viernes 07/10 en canal del "Módulo 2 - HTML" como dice @MacaCoronel93 o puedes verlo [aquí](https://web.microsoftstream.com/video/e1369afd-b280-4c18-a687-93e389a50102) (es el canal del MC en Microsoft Streams)
> 
> Gracias bro, es que me fui directo a grabaciones, pero ya me di cuenta que el link sigue activo y pueda verla desde el link. Pensaba que el link quedaba inactivo, gracias.

De nada bro! Si ese link queda activo y sino revisa mas arriba deje los canales donde están las transmisiones.

**Comment by doguedogue** [comment]
Aqui están todas las grabaciones del MC (algunas son de otras ediciones)

https://web.microsoftstream.com/user/96eeb66f-b984-4083-8ea6-74c34a07a34a

![image](https://user-images.githubusercontent.com/23409026/194793839-9a598858-60a5-4dac-9e96-f72ccc95d71b.png)


Recuerda tener la sesión de tu correo de innovaccion (por si tienes otra cuenta de MS ej.outlook) ingresa a office.com primero o hazlo en una ventana de incognito si tienes problemas
